portfolio/src/main/webapp/index.html

```diff
@@ -4,9 +4,9 @@
     <meta charset="UTF-8">
     <title>Andres's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js" defer></script>
+    <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="loadHeadFoot();loadGreeting()">
```

Comment:
Might be more readable if you created a function like onBodyLoad() and called these two load functions there instead of within the HTML

Reply:
Gotcha, thanks for the feedback. I renamed the function in script.js and changed the html accordingly.